.github/workflows/run-tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           pyvista: false
       - uses: actions/setup-python@v5
         with:
-          python-version: "3.11"  # no "multidict" wheels on 3.12 yet
+          python-version: "3.12"
```

Comment:
should be OK since v6.0.5 (2024 Feb 1) https://github.com/aio-libs/multidict/releases/tag/v6.0.5